Gems/EMotionFX/Code/Source/Editor/Plugins/ColliderWidgets/ClothJointWidget.cpp

```diff
@@ -41,14 +41,6 @@ namespace EMotionFX
         layout->setSpacing(ColliderContainerWidget::s_layoutSpacing);
         result->setLayout(layout);
 
-        // Add collider button
-        m_addColliderButton = new AddColliderButton("Add cloth collider", result,
-            PhysicsSetup::ColliderConfigType::Cloth,
-            {azrtti_typeid<Physics::CapsuleShapeConfiguration>(),
-             azrtti_typeid<Physics::SphereShapeConfiguration>()});
-        connect(m_addColliderButton, &AddColliderButton::AddCollider, this, &ClothJointWidget::OnAddCollider);
-        layout->addWidget(m_addColliderButton);
```

Comment:
this is in `    AddCollidersButton::OnCreateContextMenu()`
```
        for (const auto& section : sections)
        {
            auto& configType = section.type;
            auto* sectionItem = new QStandardItem{ section.icon, QString("Add %1 Collider").arg(section.name.data()) };

            for (auto& shape : m_supportedColliderTypes)
            {
                if (shape == azrtti_typeid<Physics::BoxShapeConfiguration>() && configType == PhysicsSetup::ColliderConfigType::Cloth)
                {
                    continue;
                }
```

Reply:
Thanks for pointing out. Could you quickly change the condition so it checks the types it does support, it's more future proof in case more shape types are added.

````
                if (configType == PhysicsSetup::ColliderConfigType::Cloth)
                {
                    const bool clothSupportedShapes = 
                        shape == azrtti_typeid<Physics::SphereShapeConfiguration>() || 
                        shape == azrtti_typeid<Physics::CapsuleShapeConfiguration>();
                    if (!clothSupportedShapes )
                    {
                        continue;
                    }
                }
````

Thanks

Reply:
Of course